### ISSUE TITLE
Sublime text 3 produced "Sublime Text has stopped working"

### ISSUE THREAD

**Issue by kisanme** [raulfraile/sublime-symfony2#30]
I use sublime text 3 with your package. When i try to enter the short code sf and hit tab, "Sublime Text has stopped working" is popped out and the program closes. My environment is Windows 8.1, Sublime Text build 3065.

**Comment by Epskampie** [comment]
I use the same build on Windows 8.1, and don't have this problem. Are you sure it's caused by this plugin? You could try a clean install with only this plugin.

**Comment by kisanme** [comment]
Yes, i did a clean install with only this plugin, the same error popped out and the program closes.

**Comment by hygull** [comment]
Follow the guide(related to delete the session data of Sublime text, it worked for my friend) available at https://kwilson.io/blog/fixing-sublime-text-has-stopped-working-on-windows-when-you-have-a-corrupted-session/ . Finally restart the computer.

**Comment by GVSHABAREESH** [comment]
when i press ctrl+b the output is not coming. just showing that [finished in 2 secs] any solution guys pls
